### PR TITLE
Style GWTP Sample Tabs

### PR DESCRIPTION
**Pull request by jasonlemay**
screen shot 2015-02-16 at 3 42 25 pm

### CODE REVIEW

File: gwtp-samples/gwtp-sample-tab/src/main/java/com/gwtplatform/samples/tab/client/application/ApplicationView.ui.xml [hunk 1]
```diff
@@ -2,20 +2,27 @@
...
-            padding: 5px;
-            font-weight: bold;
-        }
-    </ui:style>
+
+    <ui:with field="resources" type="com.gwtplatform.samples.tab.client.resources.AppResources"/>
+
     <g:HTMLPanel>
         <div style="float: right; position: relative; left: -50%;">
             <div style="position: relative; left: 50%;">
```
Comment (meriouma) [inline gwtp-samples/gwtp-sample-tab/src/main/java/com/gwtplatform/samples/tab/client/application/ApplicationView.ui.xml]: Don't want to remove those `style`?

File: gwtp-samples/gwtp-sample-tab/src/main/java/com/gwtplatform/samples/tab/client/application/homenews/HomeNewsView.java [hunk 2]
```diff
@@ -29,6 +29,7 @@
 import com.gwtplatform.samples.tab.client.resources.AppConstants;
 import com.gwtplatform.samples.tab.client.resources.AppMessages;
 import com.gwtplatform.samples.tab.client.resources.AppResources;
+import org.apache.xpath.operations.Bool;
```
Comment (meriouma) [inline gwtp-samples/gwtp-sample-tab/src/main/java/com/gwtplatform/samples/tab/client/application/homenews/HomeNewsView.java]: Remove this import

File: gwtp-samples/gwtp-sample-tab/src/main/java/com/gwtplatform/samples/tab/client/application/homenews/HomeNewsPresenter.java [hunk 3]
```diff
@@ -60,7 +60,7 @@ static TabData getTabLabel(AppConstants constants) {
      * {@link HomeNewsPresenter}'s view.
      */
     public interface MyView extends View, HasUiHandlers<HomeNewsUiHandler> {
-        void setConfirmationText(String text);
+        void setConfirmation(Boolean state);
```
Comment (meriouma) [inline gwtp-samples/gwtp-sample-tab/src/main/java/com/gwtplatform/samples/tab/client/application/homenews/HomeNewsPresenter.java]: Change to `boolean`

File: gwtp-samples/gwtp-sample-tab/src/main/java/com/gwtplatform/samples/tab/client/resources/Variables.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package com.gwtplatform.samples.tab.client.resources;
```
Comment (christiangoudreau) [inline gwtp-samples/gwtp-sample-tab/src/main/java/com/gwtplatform/samples/tab/client/resources/Variables.java]: missing licence header

File: gwtp-samples/gwtp-sample-tab/src/main/resources/com/gwtplatform/samples/tab/client/resources/css/dialog.gss [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+.dialog {
+    background-color: C_PANEL_LIGHT;
+
+    padding: 20px;
+    max-width: 500px;
+
+    -webkit-box-shadow: 0px 0px 20px 0px rgba(0,0,0,0.75);
+    -moz-box-shadow: 0px 0px 20px 0px rgba(0,0,0,0.75);
+    box-shadow: 0px 0px 20px 0px rgba(0,0,0,0.75);
+}
```
Comment (christiangoudreau) [inline gwtp-samples/gwtp-sample-tab/src/main/resources/com/gwtplatform/samples/tab/client/resources/css/dialog.gss]: missing nl at eof

File: gwtp-samples/gwtp-sample-tab/src/main/resources/com/gwtplatform/samples/tab/client/resources/css/normalize.gss [hunk 6]
```diff
@@ -0,0 +1,425 @@
+/*! normalize.css v3.0.1 | MIT License | git.io/normalize */
```
Comment (christiangoudreau) [inline gwtp-samples/gwtp-sample-tab/src/main/resources/com/gwtplatform/samples/tab/client/resources/css/normalize.gss]: We should add that in GSSS and just import it

File: gwtp-samples/gwtp-sample-tab/src/main/java/com/gwtplatform/samples/tab/client/resources/AppResources.java [hunk 7]
```diff
@@ -13,21 +25,45 @@
...
         String logo();
     }
 
-    public Styles styles();
+    @Source("css/normalize.gss")
+    Normalize normalize();
 
-    public Sprites sprites();
+    @Source({"css/variables.gss", "css/style.gss"})
+    public Style style();
```
Comment (meriouma) [inline gwtp-samples/gwtp-sample-tab/src/main/java/com/gwtplatform/samples/tab/client/resources/AppResources.java]: remove public

File: gwtp-samples/gwtp-sample-tab/src/main/java/com/gwtplatform/samples/tab/client/resources/DialogResources.java [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+import com.google.gwt.resources.client.ClientBundle;
+import com.google.gwt.resources.client.CssResource;
+
+public interface DialogResources extends ClientBundle {
+    interface Style extends CssResource {
+        String dialog();
+    }
+
+    @Source({"css/variables.gss", "css/dialog.gss"})
+    public Style style();
```
Comment (meriouma) [inline gwtp-samples/gwtp-sample-tab/src/main/java/com/gwtplatform/samples/tab/client/resources/DialogResources.java]: remove public

File: gwtp-samples/gwtp-sample-tab/src/main/java/com/gwtplatform/samples/tab/client/resources/TabsResources.java [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+        
+        String title();
+        
+        String titlebar();
+
+        String rounded();
+    }
+
+    @Source({"css/variables.gss", "css/tabs.gss"})
+    public Style style();
```
Comment (meriouma) [inline gwtp-samples/gwtp-sample-tab/src/main/java/com/gwtplatform/samples/tab/client/resources/TabsResources.java]: remove public

File: gwtp-samples/gwtp-sample-tab/src/main/resources/com/gwtplatform/samples/tab/client/resources/css/dialog.gss (hunk 5 above)
Comment (meriouma) [inline gwtp-samples/gwtp-sample-tab/src/main/resources/com/gwtplatform/samples/tab/client/resources/css/dialog.gss]: nl at EOF

**Comment by meriouma** [comment]
Am I color-blind, or is everything kinda gray? :)

File: gwtp-samples/gwtp-sample-tab/src/main/resources/com/gwtplatform/samples/tab/client/resources/css/normalize.gss (hunk 6 above)
Comment (jasonlemay) [inline gwtp-samples/gwtp-sample-tab/src/main/resources/com/gwtplatform/samples/tab/client/resources/css/normalize.gss]: @LarryMatte what do you think about adding a Reset/Normalize to GSSS?

File: gwtp-samples/gwtp-sample-tab/src/main/java/com/gwtplatform/samples/tab/client/application/ApplicationView.ui.xml (hunk 1 above)
Comment (jasonlemay) [inline gwtp-samples/gwtp-sample-tab/src/main/java/com/gwtplatform/samples/tab/client/application/ApplicationView.ui.xml]: Thing is, I don't know how to show this topMessage :/ Don't sure if it's ever used

**Comment by jasonlemay** [comment]
@meriouma both

**Pull request merged**